bigbluebutton-html5/imports/ui/components/breakout-join-confirmation/component.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { defineMessages, injectIntl } from 'react-intl';
-import { clearModal } from '/imports/ui/components/app/service';
+import { withModalMounter } from '/imports/ui/components/modal/service';
```

Comment:
Forgot to wrap the component export with the HoC. Line 74
`export default withModalMounter(injectIntl(BreakoutJoinConfirmation));`